content/guides/applications/custom-apps/jwt-setup.md

```diff
@@ -112,59 +104,60 @@ openssl rsa -in private.pem -outform PEM -pubout -out public.pem
  Windows users can install and use the [Cygwin][cygwin] package to run OpenSSL.
 </Message>
 
-Next, head over to your application in the [developer console][devconsole] and
-select the"Configuration" link from the left sidebar in your application
-and scroll down to the "Add and Manage Public Keys" section.
+Then, navigate to configuration tab for your application in the
+[Developer console][devconsole]and scroll down to the
+**Add and Manage Public Keys** section.
 
 <ImageFrame border width="600" center>
   ![Add and Manage keys](../images/app-add-keys.png)
 </ImageFrame>
 
-Click the "Add a Public Key" button to have Box upload the public key you
-created before.
+Click the **Add a Public Key** button, enter the public key generated using the
+steps above and click **Verify and Save**.
 
 ## App Authorization
 
-Once a keypair has been added to the application it will need to be authorized
-within the Box enterprise by the enterprise Admin User.
+Once a keypair is successfully added to your application your Box enterprise
+Admin needs to authorize the application within the Box Admin Console.
 
-Head over to your application in the [developer console][devconsole] and
-select the "General" link from the left sidebar in your application
-and scroll down to the "App Authorization" section.
+Navigate to the **General Settings** tab for your application within the
+[developer console][devconsole] and scroll down to the **App Authorization**
+section.
 
 <ImageFrame border width="400" center>
   ![Add and Manage keys](../images/app-authorization.png)
 </ImageFrame>
 
-This will send an email to your enterprise admin to have them enable your
-application. More information on this process is available in our [community
-article on app authorization][app-auth].
+Click **Submit and Review** to send an email to your Box enterprise Admin for
+approval. More information on this process is available in our 
+[support article for app authorization][app-auth].
 
-### Re-authorization on changes
+### Re-authorization after making configuration changes
 
 When the application's scopes or access level change the application needs to be
 re-authorized. Repeat the process above and request a new Access Token for the
 new changes to take effect.
 
 ## Basic configuration
 
-Before the application can be used, some basic additional configuration might be
+Before the application can be used, some additional, basic configuration is
```

Comment:
This reads a little strange to me (the old sentence does too). Perhaps just cut it down to "Before the application can be used, some additional configuration is required".